setup.py

```diff
@@ -6,15 +6,22 @@
 setuptools.setup(
     name="tsod",
     version="0.2.0",
-    install_requires=["pandas>=1.0.0", "numba", "joblib"],
+    install_requires=[
+        "pandas==2.1.*",
```

Comment:
Don't pin versions here.

`tsod` is a library intended to use together with other libraries. Pinning versions will make this very difficult.